docs/Cloud_integrations/Device_Integration_Center/overview.md

```diff
@@ -114,6 +114,22 @@ However, in the `abc` mode, variables can also be used with the help of the `$`
       * `domainId`
       * `domainName`
 
+### Forbidden headers
+
+Please be aware that certain headers cannot be overwritten when using a webhook. 
```

Comment:
Fine, tho PLEASE don't use "please" in the documentation :) We don't need to be polite - I mean there is no reason for using it, just be precise and concrete. So rephrase as follows:
"Certain headers can't be overwritten when using a webhook. 
The following list includes headers that are restricted and can't be modified:"